Collectors/DetailedCollector.py

```diff
@@ -374,17 +468,18 @@ def process(self, data, addr, port):
             pass
 
         elif header.code == b'g':
-            # The rest of the message is the gstream event
-            self.logger.debug("Received gstream message")
-            decoded_gstream = decoding.gStream(data)
-
-            # We only care about the top 8 bits of the ident, which are a character.
-            stream_type = chr(decoded_gstream.ident >> 56)
-            if stream_type == "T":
-                self.process_tcp(decoded_gstream, addr)
-            print(decoded_gstream)
-
-
+        
+         # The rest of the message is the gstream event
+         self.logger.debug("Received gstream message")
+         decoded_gstream = decoding.gStream(data)
+
+         # We only care about the top 8 bits of the ident, which are a character.
+         stream_type = chr(decoded_gstream.ident >> 56)
+         if stream_type == "T":
+              self.process_tcp(decoded_gstream, addr)
+         
+         # process the gstream
+         self.process_gstream(decoded_gstream, addr,sid)
```

Comment:
Why is this shifted to the left?  Shouldn't it be in the `elif header.code == b'g':`

Reply:
should be on `elif header.code == b'g':` maybe I just changed the positions.

Reply:
Certainly doesn't look like it in the diff.

Reply:
ops, wrong file :-)